lib/cisco_node_utils/bridge_domain.rb

```diff
@@ -93,7 +93,7 @@ def self.bds
           bd_list.map { |elem| BridgeDomain.bd_ids_to_array(elem) }
           .flatten.uniq.sort
       final_bd_list.each do |id|
-        hash[id] = BridgeDomain.new(id, false)
+        hash[id.to_s] = BridgeDomain.new(id, false)
```

Comment:
Wanted to keep all the return values consistent with other getters in this node util class, all the getter return string values.